tests/browser/pages/relying-party.js

```diff
@@ -1,39 +1,66 @@
+const axios = require("axios");
+
 module.exports = class PlaywrightDevPage {
   /**
    * @param {import('@playwright/test').Page} page
    */
   constructor(page, clientId) {
     this.page = page;
-    this.clientId = clientId;
 
-    // Starting URL properties
     const websiteHost = process.env.WEBSITE_HOST || "http://localhost:5050";
     this.baseURL = new URL(websiteHost);
+    this.relyingPartyURL = new URL(process.env.RELYING_PARTY_URL);
+    this.env = process.env.ENV || "dev";
 
-    this.oauthPath = `/oauth2/authorize?request=lorem&client_id=${this.clientId}`;
-    this.startingURL = new URL(this.oauthPath, this.baseURL);
-
-    // Relying Party Return URL
-    this.relyingPartyURL = new URL("http://example.net");
+    if (process.env.MOCK_API === "true") {
```

Comment:
I suppose the presence of MOCK_API env variable, irrespective of value should mean this is a mock. My reasoning here is the behaviour of environment variables in general; they are not required by nature. However, we are implying that `MOCK_API` is required

Reply:
Hmm yeah we wanted to be able to explicitly toggle it since env vars are strings.

As is, if you don't set `MOCK_API` it will default to `true` which seems strange based on the name but made most sense in the context - if you haven't set `MOCK_API` then it's likely you want to use the defaults. To use the defaults we either need to use the local imposter mock or we need to point you to an environment, then how do we decide which environment?

I wonder if renaming to something like `USE_LOCAL_API` might be more clear?

Reply:
Yeah, I was beginning to think the same `USE_LOCAL_API` would be the ideal